src/docs/transactions_api.dox

```diff
@@ -63,11 +63,17 @@ other writes, although concurrent transactions updating the same value will fail
 synchronization to avoid repeated attempts to update and then roll back the same
 value.
 
+Transactions hold resources, such as cache space, that may be in high demand.
 Both read and write transactions may fail with the ::WT_ROLLBACK error if a
 resource cannot be allocated. For example, if cache resources cannot be
 allocated to hold the data required to satisfy transactional readers, operations
 may fail and return ::WT_ROLLBACK.
 
+\warning
+A thread with active transactions should avoid pausing or blocking. This may delay
```

Comment:
Since a thread is using one session (and therefore one transaction) I'd say `A thread with an active transaction...`